src/helpers/validation_schema.js

```diff
@@ -1,23 +1,32 @@
-// import Joi from "joi";
+import Joi from "joi";
 
+export const registerValidation = (user) => {
+  const schema = Joi.object({
+    firstName: Joi.string().min(5).required(),
+    lastName: Joi.string().min(5).required(),
+    role: Joi.string().min(5).required(),
+    email: Joi.string().min(5).required().email(),
+    password: Joi.string().min(5).required(),
+  });
+  return schema.validate(user);
+}
 
-// export const UserValidation=(data)=>{
```

Comment:
please after creating a new function for certain data please remember also to remove the commented codes